tests/k8s/test_config_reload.py

```diff
@@ -24,9 +25,9 @@ def test_config_reload_no_toggle(duthost, k8scluster):
         k8scluster: shortcut fixture for getting cluster of Kubernetes master hosts
     """
     ku.join_master(duthost, k8scluster.vip) # Assertion within to ensure successful join
-    dut_cmds = ['sudo config save -y',
-                'sudo config reload -y']
+    dut_cmds = ['sudo config save -y']
```

Comment:
do you still need to have it like that for single command?

Reply:
Modified it